tests/templates/kuttl/tls/10-install-zookeeper.yaml.j2

```
@@ -0,0 +1,48 @@
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperCluster
+metadata:
+  name: test-zk
+spec:
+  version: {{ test_scenario['values']['zookeeper'] }}
+  config:
+{% if test_scenario['values']['use-client-tls'] == 'true' %}
+    tls:
+      secretClass: tls
+    clientAuthentication:
+      authenticationClass: zk-client-tls
+{% else %}
+    tls: null
+{% endif %}
+{% if test_scenario['values']['use-internal-tls'] == 'true' %}
+    internalTlsSecretClass: tls
+{% endif %}
```

Currently connection Kafka to tls enabled zk is not supported 
```suggestion
  config:
    tls: null
```